tern/analyze/default/container/multi_layer.py

```diff
@@ -67,6 +67,10 @@ def fresh_analysis(image_obj, curr_layer, shell, driver, envs):
                     envs)
         rootfs.undo_mount()
         rootfs.unmount_rootfs()
+    # distroless images do not have a command list
```

Comment:
I don't think there's any way of collecting info from the things that get added on top of the first layer. My investigation has found that `/var/lib/dpkg/status.d` doesn't get updated in the second layer for all the distroless images besides the static one. Let's leave this change out for now until we can figure out an alternative.

Reply:
This is not the experience that I found. When scanning `gcr.io/distroless/dotnet:latest` I find 4 layers, each with packages:

<details>
<summary>Details</summary>

```
Docker image: gcr.io/distroless/dotnet@sha256:8a4099dee4df6f856bad39bbcf598255fa30061cd8227bbd08edda147999f54b:
        Layer 1:
                warning: No known shell found in image. Cannot invoke commands

                info: Found 'Distroless' in /etc/os-release.
                warning: No listing method for 'files'. Additional analysis may be required.
No listing method for 'licenses'. Additional analysis may be required.
No listing method for 'srcs'. Additional analysis may be required.

        File licenses found in Layer:  None
        Packages found in Layer:  base-files-9.9+deb9u13, netbase-5.4, tzdata-2021a-0+deb9u1
        Licenses found in Layer:  None
------------------------------------------------

        Layer 2:
                warning: No created_by information for layer.
                warning: No listing method for 'files'. Additional analysis may be required.
No listing method for 'licenses'. Additional analysis may be required.
No listing method for 'srcs'. Additional analysis may be required.

        File licenses found in Layer:  None
        Packages found in Layer:  libc6-2.24-11+deb9u4, libssl1.1-1.1.0l-1~deb9u3, openssl-1.1.0l-1~deb9u3
        Licenses found in Layer:  None
------------------------------------------------

        Layer 3:
                warning: No created_by information for layer.
                warning: No listing method for 'files'. Additional analysis may be required.
No listing method for 'licenses'. Additional analysis may be required.
No listing method for 'srcs'. Additional analysis may be required.

        File licenses found in Layer:  None
        Packages found in Layer:  libgcc1-1:6.3.0-18+deb9u1, libgomp1-6.3.0-18+deb9u1, libstdc++6-6.3.0-18+deb9u1
        Licenses found in Layer:  None
------------------------------------------------

        Layer 4:
                warning: No created_by information for layer.
                error: /bin/cat: /usr/share/doc/libcomerr2/copyright: No such file or directory
/bin/cat: /usr/share/doc/libcurl3/copyright: No such file or directory
/bin/cat: /usr/share/doc/libicu57/copyright: No such file or directory
/bin/cat: /usr/share/doc/libidn11/copyright: No such file or directory
/bin/cat: /usr/share/doc/libssl1.0.2/copyright: No such file or directory
/bin/cat: /usr/share/doc/libunistring0/copyright: No such file or directory

                warning: No listing method for 'files'. Additional analysis may be required.
No listing method for 'licenses'. Additional analysis may be required.
No listing method for 'srcs'. Additional analysis may be required.

        File licenses found in Layer:  None
        Packages found in Layer:  curl-7.52.1-5+deb9u13, libacl1-2.2.52-3+b1, libattr1-1:2.4.47-2+b2, libbz2-1.0-1.0.6-8.1, libcomerr2-1.44.5-1~bpo9+1, libcurl3-7.52.1-5+deb9u13, libdb5.3-5.3.28-12+deb9u1, libffi6-3.2.1-6, libgcrypt20-1.7.6-2+deb9u3, libgmp10-2:6.1.2+dfsg-1, libgnutls30-3.5.8-5+deb9u5, libgpg-error0-1.35-1~bpo9+1, libgssapi-krb5-2-1.15-1+deb9u2, libhogweed4-3.3-1+b2, libicu57-57.1-6+deb9u4, libidn11-1.33-1+deb9u1, libidn2-0-2.0.5-1~bpo9+1, libk5crypto3-1.15-1+deb9u2, libkeyutils1-1.5.9-9, libkrb5-3-1.15-1+deb9u2, libkrb5support0-1.15-1+deb9u2, libldap-2.4-2-2.4.47+dfsg-3+deb10u2~bpo9+1, liblttng-ust0-2.9.0-2+deb9u1, liblzma5-5.2.2-1.2+b1, libnettle6-3.3-1+b2, libnghttp2-14-1.18.1-1+deb9u1, libp11-kit0-0.23.3-2+deb9u1, libpcre3-2:8.39-3, libpsl5-0.17.0-3, librtmp1-2.4+20151223.gitfa8646d.1-1+b1, libsasl2-2-2.1.27~101-g0780600+dfsg-3+deb9u1, libselinux1-2.6-3+b3, libssh2-1-1.7.0-1+deb9u1, libssl1.0.2-1.0.2u-1~deb9u4, libtasn1-6-4.10-1.1+deb9u1, libunistring0-0.9.6+really0.9.3-0.1, libunwind8-1.1-4.1, libuuid1-2.29.2-1+deb9u1, zlib1g-1:1.2.8.dfsg-5
        Licenses found in Layer:  None
------------------------------------------------
```

</details>